pkgs/development/python-modules/imantics/default.nix

```
@@ -0,0 +1,43 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, numpy
+, opencv3
+, sphinx_rtd_theme
+, lxml
+, xmljson
+}:
+
+buildPythonPackage rec {
+  pname = "imantics";
+  version = "0.1.12";
+
+  src = fetchFromGitHub {
+    owner = "jsbroks";
+    repo = "imantics";
+    rev = "76d81036d8f92854d63ad9938dd76c718f8b482e";
+    sha256 = "1zv2gj8cbakhh2fyr2611cbqhfk37a56x973ny9n43y70n26pzm8";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    opencv3
+    sphinx_rtd_theme
+    lxml
+    xmljson
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'opencv-python>=3'," ""
+  '';
+
+  doCheck = False;
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```

This fixes the eval error.

---

This should not be an issue as i have imported lib on top of the package.

---

oh  sorry, I misread it. correcting